examples/ct/structures_test.go

```diff
@@ -43,4 +58,54 @@ func TestGetCTLogIDNotLoaded(t *testing.T) {
 }
 
 func TestSignV1SCTForCertificate(t *testing.T) {
+	cert, err := fixchain.CertificateFromPEM(testonly.LeafSignedByFakeIntermediateCertPem)
```

Comment:
I'm reusing it for some of the checks it does on the certificates to avoid re-implementing them. I do an additional check that there's a chain using all the submitted certs in order.